Update client contract tutorial (#2160)
filecoin-project/filecoin-docs: content/en/smart-contracts/developing-contracts/client-contract-tutorial/index.md

Now if you’ve invoked the task with all the correct parameters, the method will execute on-chain and emit an event that Boost storage providers will be listening to. If the deal is well-formed and the parameters are acceptable, they will download the .car file, double-check to ensure the `piece-cid` and `piece-size` match the deal and publish your storage deal! This could take up to a day. Once the deal is published, you can find it on a [Calibration testnet block explorer]({{< relref "/networks/calibration/explorers" >}}). The client in the deal should be the `t4` address of the smart contract we called `MakeStorageDeal` on.

### Monitoring deal proposal acceptance

> After emitting an event and while waiting for Storage Providers to accept your deal, you can monitor its status on a provided Boost logs dashboard (for calibration testnet only). You can see a guide [here](https://github.com/filecoin-project/community/discussions/659) on how to diagnose why deals might not be being accepted and adjust your proposal for re-submission.

Reviewer: ```suggestion
After emitting an event and waiting for storage providers to accept your deal, you can monitor its status on a provided Boost logs dashboard. This feature is only available on the Calibration testnet. [See this guide on GitHub](https://github.com/filecoin-project/community/discussions/659) for help diagnosing why deals might not be accepted and adjusting your proposal for re-submission.
```
